### PR TITLE
Add script to delete SPV contents.

### PR DESCRIPTION
**Pull request by njohner**
A script to delete all SPV contents. I hope I did not forget anything. I tested it locally.

### CODE REVIEW

File: opengever/maintenance/scripts/deactivate_spv.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            for brain in self.catalog.unrestrictedSearchResults({'portal_type': portal_type}):
+                obj = brain.getObject()
+                print("deleting {}".format(obj.absolute_url()))
+                aq_parent(aq_inner(obj)).manage_delObjects([obj.getId()])
+
+    def clear_sql_tables(self):
+        print("Starting deletion of tables")
+        metadata = MetaData(self.connection, reflect=True)
+        for table_name in (
+                'agendaitems',  # => proposals, meetings
```
Comment (deiferni) [inline opengever/maintenance/scripts/deactivate_spv.py]: i think you should add `'generateddocuments'` and `'meetings'`

Comment (njohner) [reply opengever/maintenance/scripts/deactivate_spv.py]: Yes, sorry, I misread an upgrade step which I thought deleted the table... But no, it's still here

File: opengever/maintenance/scripts/deactivate_spv.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    if not len(args) == 0:
+        print "Not expecting any argument"
+        sys.exit(1)
+
+    if options.dryrun:
+        print "dry-run ..."
+        transaction.doom()
+
+    delete_tables = options.delete_tables
+    cleaner = CleanupSPVAndEnableWordFeature(dry_run=options.dryrun)
```
[the remark this answers is not in the document]
Comment (njohner) [reply opengever/maintenance/scripts/deactivate_spv.py]: Yes. Don't even dare explain my reasoning why I took them out... but at second thought you're right!

File: opengever/maintenance/scripts/deactivate_spv.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    def remove_plone_content(self):
+        portal_types = (
+            'opengever.meeting.committee',
+            'opengever.meeting.committeecontainer',
+            'opengever.meeting.period',
+            'opengever.meeting.submittedproposal',
+            'opengever.meeting.proposal',
+            'opengever.meeting.proposaltemplate',
+            'opengever.meeting.sablontemplate',
+            'opengever.meeting.meetingdossier',
```
[the remark this answers is not in the document]
Comment (njohner) [reply opengever/maintenance/scripts/deactivate_spv.py]: Yes. Don't even dare explain my reasoning why I took them out... but at second thought you're right!

**Comment by njohner** [comment]
Script was run successfully for `http://ris.onegovgever.ch/kr` and `http://ris.onegovgever.ch/rr`

**Pull request merged**